src/repositories/DocumentCategoryRepository.ts

```diff
@@ -14,7 +14,7 @@ export const getDocumentCategoriesByDocumentId = async (documentId: number): Pro
 
 const mapDocumentCategoryApiToFrontifyDocumentCategory = (
```

Comment:
We should update method name as well:
```suggestion
const mapDocumentCategoryApiToDocumentCategory = (
```